src/styles/blocks/card.scss

```diff
@@ -8,10 +8,8 @@
   border: 1px solid #f3f3f3;
   border-radius: 5px;
 
-  &--margin-right-bottom {
-    margin-right: $marginRightCard;
-    margin-bottom: 46px;
-  }
+  margin-right: 48px;
+  margin-bottom: 46px;
```

Comment:
I would actually create some wrapper containing your block for that, for not influencing the outer environment